swift/ios-shoppe-demo/Views/ProductCollectionViewCell.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  ProductCollectionViewCell.swift
+//  ios-shoppe-demo
+//
+//  Created by Harold Davis Jr. on 3/25/20.
```

Comment:
The header has your name versus others that have FullStory

Reply:
Ooh yes absolutely thanks for the catch!